third_party/tlx/language/tlx/__init__.py

```diff
@@ -1,76 +1,71 @@
+from . import compiler
 from .async_task_utils import async_task, async_tasks
-from .types import (
-    layout_encoding,
-    shared_layout_encoding,
-    swizzled_shared_layout_encoding,
-    tensor_memory_layout_encoding,
-    nv_mma_shared_layout_encoding,
-    storage_kind,
-    buffered_tensor,
-    buffered_tensor_type,
-    mbarrier,
-    mbarrier_type,
-    clc_response,
-    clc_response_type,
-    CLCPipelineContext,
-    async_token,
-    tensor_descriptor_ptr,
-    tensor_descriptor_ptr_type,
+from .barrier import (
```

Comment:
This was done by vscode automatically. Since this is an internal file I think we should keep this reordering.